docs/content/examples/alt-text.html

```diff
@@ -0,0 +1,152 @@
+    </p>
+    <div class="ta-center w100 wmx5 mb8">
+        <img src="{{ "/assets/img/community.svg" | relative_url }}"" alt="Man and woman using laptops at a standing table, Illustration.">
+    </div>
+    <table class="s-table s-table__lg va-middle fs-body2 va-top w100 wmx5 mb48">
+        <thead>
+            <th class="btw0 bxw0 bbw3 bbw3 bc-red-2 fc-red-500 bg-transparent fs-body1 s-table--cell6">{% icon Clear | mtn1 mbn1 %} Don’t</th>
+            <th class="btw0 bxw0 bbw3 bc-green-2 fc-green-500 bg-transparent fs-body1 s-table--cell6">{% icon Checkmark | mtn1 mbn1 %} Do</th>
+        </thead>
+        <tbody>
+            <tr>
+                <td>“People using computers.”</td>
+                <td rowspan="3" class="bb bc-black-3">“Man and woman using laptops at a standing table, Illustration.”</td>
+            </tr>
+            <tr>
+                <td>“Man in a purple shirt and woman in blue pants are typing on laptops and there’s a plant on the floor nearby.”</td>
+            </tr>
+            <tr>
+                <td>“Man and woman using computers, illustrated by Jane Doe © 2019.”</td>
+            </tr>
+        </tbody>
+    </table>
+    <p class="stacks-copy">
+        Take context into account. For instance, if the image above is part of a blog post about standing tables, then it’s safer to skip the part about standing tables.
+    </p>
+    {% header h3 | Don’t say it’s an image %}
+    <p class="stacks-copy">
+        Don’t start alt-texts with things like “Image of” or “Photo of.”. Screen readers add that by default. If it’s a special type of image (like an icon), you can note that at the end.
+    </p>
+    <table class="s-table s-table__lg va-middle fs-body2 va-top w100 wmx5 mb48">
+        <thead>
+            <th class="btw0 bxw0 bbw3 bbw3 bc-red-2 fc-red-500 bg-transparent fs-body1 s-table--cell6">{% icon Clear | mtn1 mbn1 %} Don’t</th>
+            <th class="btw0 bxw0 bbw3 bc-green-2 fc-green-500 bg-transparent fs-body1 s-table--cell6">{% icon Checkmark | mtn1 mbn1 %} Do</th>
+        </thead>
+        <tbody>
+            <tr>
+                <td>“Image of a rocket.”</td>
+                <td rowspan="3">“A rocket.”</td>
+            </tr>
+            <tr>
+                <td>“Illustration of a rocket.”</td>
+            </tr>
+            <tr>
+                <td>Phoot of a rocket.”</td>
+            </tr>
+            <tr>
+                <td>“Icon of a rocket.”</td>
+                <td>“A rocket, icon.”</td>
+            </tr>
+        </tbody>
+    </table>
+    {% header h3 | End with a period %}
+    <p class="stacks-copy">
+        End the alt-text with a period. This makes screen readers pause a bit after the last word in the alt-text, creating a natural pause before the next bit of text.
+    </p>
+</section>
+<section class="stacks-section">
+    {% header h2 | When not to write one %}
```

Comment:
"When not to write one" doesn't seem to match the tone in the other headings.

Maybe "When not to include alt-text"?